### ISSUE TITLE
Can't set more than one encrypter/decrypter for a key

### ISSUE THREAD

**Issue by muffl0n** [terraform-google-modules/terraform-google-kms#73]
### TL;DR

Setting more than one entity in `encrypters` and `decrypters` does not work. The other entities other than the first get silently ignored.

### Expected behavior

All entities in `encrypters` and `decrypters` are given the roles `roles/cloudkms.cryptoKeyEncrypter` and `roles/cloudkms.cryptoKeyDecrypter`

### Observed behavior

Adding a second entity to `encrypters` and `decrypters` does nothing.

### Terraform Configuration
This adds the roles `roles/cloudkms.cryptoKeyEncrypter` and `roles/cloudkms.cryptoKeyDecrypter` to the service account `google_service_account.kanister`.

```hcl
module "kms" {
  source  = "registry.terraform.io/terraform-google-modules/kms/google"
  version = "2.2.1"

  project_id = var.project.name
  location   = "europe"
  keyring    = "backup"
  keys       = ["kanister"]

  encrypters = [
    "serviceAccount:${google_service_account.kanister.email}",
  ]
  decrypters = [
    "serviceAccount:${google_service_account.kanister.email}",
  ]

  set_encrypters_for = ["kanister"]
  set_decrypters_for = ["kanister"]
}
```

Adding a secondary account does nothing:
```hcl
module "kms" {
  source  = "registry.terraform.io/terraform-google-modules/kms/google"
  version = "2.2.1"

  project_id = var.project.name
  location   = "europe"
  keyring    = "backup"
  keys       = ["kanister"]

  encrypters = [
    "serviceAccount:${google_service_account.kanister.email}",
    "serviceAccount:service-2342@gs-project-accounts.iam.gserviceaccount.com",
  ]
  decrypters = [
    "serviceAccount:${google_service_account.kanister.email}",
    "serviceAccount:service-2342@gs-project-accounts.iam.gserviceaccount.com",
  ]

  set_encrypters_for = ["kanister"]
  set_decrypters_for = ["kanister"]
}
```

### Terraform Version

```sh
Terraform v1.3.5
on darwin_amd64
+ provider registry.terraform.io/hashicorp/google v4.44.1
+ provider registry.terraform.io/hashicorp/google-beta v4.44.1
+ provider registry.terraform.io/hashicorp/kubernetes v2.16.0
+ provider registry.terraform.io/hashicorp/random v3.4.3
```

**Comment by muffl0n** [comment]
Adding `kanister` a second time to `set_encrypters_for` and `set_decrypters_for` seems to do the trick. This feels quite cumbersome. 
Is there a part of the documentation I'm missing?
